reactor-rxjava-guide.md

```diff
@@ -0,0 +1,119 @@
+# Reactor vs RxJava guide
+
+The purpose of this guide is to help those who are more familiar with the RxJava framework to familiarize themselves with the Reactor framework and Azure Cosmos DB Java SDK 4.0 for Core (SQL) API ("Java SDK 4.0" from here on out.)
+
+Users of Async Java SDK 2.x.x should read this guide to understand how familiar async tasks can be performed in Reactor. We recommend first reading the [Reactor pattern guide](reactor-pattern-guide.md) for more general Reactor introduction.
+
+A quick refresher on Java SDK versions:
+
+| Java SDK                | Release Date | Bundled APIs         | Maven Jar                               | Java package name                |API Reference                                              | Release Notes                                                                            |
+|-------------------------|--------------|----------------------|-----------------------------------------|----------------------------------|-----------------------------------------------------------|------------------------------------------------------------------------------------------|
+| Async 2.x.x             | June 2018    | Async(RxJava)        | com.microsoft.azure::azure-cosmosdb     | com.microsoft.azure.cosmosdb.rx  | [API](https://azure.github.io/azure-cosmosdb-java/2.0.0/) | [Release Notes](https://docs.microsoft.com/en-us/azure/cosmos-db/sql-api-sdk-async-java) |
+| "Legacy" Sync 2.x.x     | Sept 2018    | Sync                 | com.microsoft.azure::azure-documentdb   | com.microsoft.azure.cosmosdb     | [API](https://azure.github.io/azure-cosmosdb-java/2.0.0/) | [Release Notes](https://docs.microsoft.com/en-us/azure/cosmos-db/sql-api-sdk-java)       |
```

Comment:
legacy sync v2 package name is not `com.microsoft.azure.cosmosdb`. change to `com.microsoft.azure.documentdb`